### PR TITLE
Paramedic Autoinjector + Medical QOL

### PR DESCRIPTION
**Pull request by a-person5660**
## About the PR
gives the paramedics a cartridge autoinjector round start, three extra cartridges and a tourniquet.
puts tourniquets in the medical vendors and allows them to be placed in medical belts.
adds medical cartridge autoinjector to the medfab.
slightly decreases cartridge autoinjector recharge time to a round number for my own sanity.

## Why / Balance
https://discord.com/channels/1202734573247795300/1313408235478323230 
also its a cool idea.

**Changelog**
:cl:
- add: Added a medical cartridge autoinjector.
- tweak: Cartridge autoinjectors now have a 30 second cooldown.
- tweak: EMT belts now contain a tourniquet and medical cartridge autoinjector.
- tweak: Medical vendors now contain six tourniquets and two medical cartridge autoinjectors.
- tweak: Ichor is no longer printable at an emagged autolathe, in its place is puncturase cartridges.
- fix: Tourniquets can now go inside medical belts.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Recipes/Lathes/medical.yml [hunk 1]
```diff
@@ -27,3 +27,13 @@
...
+
+- type: latheRecipe
+  id: ParamedHypo
+  result: ParamedHypo
+  completetime: 3
+  materials:
+    Steel: 1500
+    Plastic: 800
+    Glass: 1200
+    Gold: 500
```
[the remark this answers is not in the document]
Comment (a-person5660) [reply Resources/Prototypes/_Goobstation/Recipes/Lathes/medical.yml:39]: you make an amazing point, tbh the syringe gun should be more expensive for what it is.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 2]
```diff
@@ -870,7 +871,7 @@
       - CartridgeBicaridine # Goobstation
       - CartridgeDermaline # Goobstation
       - SecHypo # Goobstation
-      - SyringeGun # Goobstation  
+      - SyringeGun # Goobstation
```
[the remark this answers is not in the document]
Comment (a-person5660) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:874]: oops

File: Resources/Prototypes/_Goobstation/Recipes/Lathes/medical.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (a-person5660) [reply Resources/Prototypes/_Goobstation/Recipes/Lathes/medical.yml:39]: changing the syringe guns price is out of the scope of this PR and i think the price is good enough and the syringe guns should be upped later.

**Comment by JoeHammad1844** [comment]
Minor spelling mistake in the CL
You typed not instead of now

**Comment by Aviu00** [comment]
Can you also add ephedrine and ichor cartridges to emagged medfab, there is no reason for them not to be there.

**Comment by a-person5660** [comment]
> Can you also add ephedrine and ichor cartridges to emagged medfab, there is no reason for them not to be there.

tried but it gave me a yaml fail so i didnt bother.

**Comment by a-person5660** [comment]
realised that ichor is a bit goated, its been replaced by puncturase :trollface:

**Comment by JoeHammad1844** [comment]
okay this is epic

**Pull request merged**